packages/frontend/web-ui/src/auth/pages/Login.spec.tsx

```diff
@@ -225,29 +224,30 @@ describe(Login.name, () => {
 
   describe('when called, and user exists and navigate to the next page', () => {
     beforeAll(() => {
-      useNavigate.mockReturnValue(navigateMock);
+      tokenFixture = 'jwt token fixture';
+
+      (useNavigate as jest.Mock<typeof useNavigate>).mockReturnValue(
+        navigateMock,
+      );
 
-      useLoginForm.mockReturnValue({
+      (useLoginForm as jest.Mock<typeof useLoginForm>).mockReturnValue({
         backendError: null,
         formFields: formFieldsFixture,
-        formStatus: STATUS_LOG_BACKEND_OK,
+        formStatus: LoginStatus.backendOK,
         formValidation: {},
         notifyFormFieldsFilled: notifyFormFieldsFilledMock,
         setFormField: setFormFieldMock,
       });
 
-      useShowPassword.mockReturnValue({
+      (useShowPassword as jest.Mock<typeof useShowPassword>).mockReturnValue({
         showPassword: false,
         handleClickShowPassword: handleClickShowPasswordMock,
         handleMouseDownPassword: handleMouseDownPasswordMock,
       });
 
-      tokenFixture = 'jwt token fixture';
-
-      useSelector.mockImplementation(() => ({
-        token: tokenFixture,
-        errorMessage: errorMessageFixture,
-      }));
+      (
+        useAppSelector as unknown as jest.Mock<typeof useAppSelector>
+      ).mockReturnValueOnce(tokenFixture);
 
       render(
         <MemoryRouter>
```

Comment:
The `afterAll` block is missing the reset of `useLoginForm`, `useShowPassword`

Reply:
Done!